airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -610,7 +610,15 @@ def delete_objects(self, bucket, keys):
         if isinstance(keys, str):
             keys = [keys]
 
-        delete_dict = {"Objects": [{"Key": k} for k in keys]}
-        response = self.get_conn().delete_objects(Bucket=bucket, Delete=delete_dict)
-
-        return response
+        s3 = self.get_conn()
+        batch = 1000
+        for i in range(0, len(keys), batch):
```

Comment:
Why do you think about using airflow.helpers.chunks here?

Reply:
I didn't know that one existed. Thanks! :)